docs/sources/flow/getting-started/run-grafana-agent-flow.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Run Grafana Agent Flow
+weight: 100
+---
+
+# Run Grafana Agent Flow
+
+This topic describes how to run Grafana Agent Flow.
+
+## Before you begin
+
+* [Install Grafana Agent][]
+
+[Install Grafana Agent]: {{< relref "../../set-up" >}}
+
+## Steps
+
+To run Grafana Agent Flow, follow these steps:
+
+1. Set the `AGENT_MODE` environment variable on your system to `flow`.
+
+2. Create a configuration file to Grafana Agent Flow to use, saving it to your
+   system. For example:
+
+   ```river
```

Comment:
should we mention somewhere on this page the config file is a `.river`?

Reply:
Sure, I'll do that 👍